Assets/MixedRealityToolkit.Extensions/MarkerDetection/IVariableSizeMarkerDetector.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+namespace Microsoft.MixedReality.Toolkit.Extensions.Experimental.MarkerDetection
+{
+    public interface IVariableSizeMarkerDetector : IMarkerDetector
+    {
+        /// <summary>
+        /// Tries to obtain the physical size of a detected marker.
+        /// </summary>
+        /// <param name="markerId">id of detected marker</param>
+        /// <param name="size">The physical size (in meters) of the detected marker</param>
+        /// <returns>True if the size was located</returns>
+        bool TryGetMarkerSize(int markerId, out float size);
```

Comment:
Why not update IMarkerDetector with this?

Reply:
Hm, so I think there are two types of marker detectors that will have different usage:

1) Marker detectors that need to know the exact physical size of the marker
2) Marker detectors that can determine the exact physical size of the marker

I think we should have clear abstractions for the two types of detectors. I don't think that overloading the same interface with this functionality makes usage very clear. However, it may also be worth moving the setsize function in IMarkerDetector into an IStaticSizeMarkerDetector.cs interface

Reply:
moved to a single interface for usability